python/_nimblephysics/biomechanics/SubjectOnDisk.cpp

```diff
@@ -205,6 +206,14 @@ Note that these are specified in the local body frame, acting on the body at its
 
             This method is provided to give a cheaper way to filter out frames we want to ignore for training, without having to call
             the more expensive :code:`loadFrames()`
+          )doc")
```

Comment:
I think you also need to add an explicit PyBind11 enum declaration for your new binding, if you don't want it to blow up when you return from this. Docs here: https://pybind11.readthedocs.io/en/stable/classes.html#enumerations-and-internal-types

Reply:
Okay, I attempted wrapping the enum, but I'm not exactly sure the best way to test it.

Reply:
Okay, should work now. The pybind stub regeneration worked.